test/e2e/install/install_test.go

```diff
@@ -350,7 +350,7 @@ func overallDeployment() nroDeployment {
 		mcpObj := objects.TestMCP()
 		By(fmt.Sprintf("creating the machine config pool object: %s", mcpObj.Name))
 		err := e2eclient.Client.Create(context.TODO(), mcpObj)
-		Expect(err).NotTo(HaveOccurred())
+		ExpectWithOffset(1, err).NotTo(HaveOccurred())
```

Comment:
We can always examine the call stack and find the caller function but I guess this change won't hurt.

Reply:
my rationale was poorly written, hopefully this helps more: https://onsi.github.io/gomega/#making-assertions-in-helper-functions

Reply:
I understood the rationale. 
But I was wrong, we're not getting the call stack, only the spec's line, so nice trick indeed!